src/Features/Core/Portable/ChangeSignature/AddedParameterResult.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ChangeSignature
+{
+    internal sealed class AddedParameterResult
+    {
+        public AddedParameter AddedParameter;
+        public bool IsCancelled { get; set; }
```

Comment:
1. what does IsCanceled mean?
2. why is one a mutable field, and one a mutable property?

Reply:
removed this class at all

---
In reply to: [367214548](https://github.com/dotnet/roslyn/pull/38748#discussion_r367214548) [](ancestors = 367214548)